Block/Product.php

```diff
@@ -52,8 +53,7 @@
 use Nosto\Tagging\Helper\Account as NostoHelperAccount;
 use Nosto\Tagging\Helper\Scope as NostoHelperScope;
 use Nosto\Tagging\Model\Category\Builder as NostoCategoryBuilder;
-use Nosto\Tagging\Model\Product\Builder as NostoProductBuilder;
-use Nosto\Tagging\Model\Service\Product as NostoProductService;
+use Nosto\Tagging\Model\Service\ProductServiceInterface as NostoProductService;
```

Comment:
Why alias this? I'd say it's more readable and consistent with other interface injections if you leave it as `ProductServiceInterface`